auth0_flutter/lib/src/web/js_interop_utils.dart

```diff
@@ -22,7 +22,9 @@ class JsInteropUtils {
   // **Note**: there is no static typing for these parameters to be able
   // to retrieve them again.
   static T addCustomParams<T extends Object>(
-      final T obj, final Map<String, dynamic> params) {
+    final T obj,
+    final Map<String, dynamic> params,
```

Comment:
Adding a comma here was needed for the formatter to properly format this under 80 characters per line.